hbase_sql/pom.xml

```diff
@@ -297,7 +297,12 @@
             <version>${spark.version}</version>
             <scope>provided</scope>
         </dependency>
-
+        <dependency>
+            <groupId>org.apache.hive</groupId>
+            <artifactId>hive-classification</artifactId>
+            <version>${hive.version}</version>
+            <scope>provided</scope>
+        </dependency>
```

Comment:
sure we need this additional dependency?

Reply:
org.apache.hadoop.hive.serde2.SerDe is replaced by org.apache.hadoop.hive.serde2.AbstractSerDe. It needs this dependency for Public annotation to compile.

Reply:
ok

Reply:
Add a comment to the dependency mentioning the required annotation to help future developers